packages/vsce/src/api/SshMvsApi.ts

```diff
@@ -20,6 +20,7 @@ export class SshMvsApi extends SshCommonApi implements MainframeInteraction.IMvs
         try {
             const response = await (await this.client).ds.listDatasets({
                 pattern: filter,
+                attributes: true,
             });
```

Comment:
Should we pass the attributes option provided by the API caller instead of hardcoding `true`?